commandlineparser/include/Options.h

```diff
@@ -0,0 +1,29 @@
+#include<iostream>
+#include<string>
+
+//stores value of various command options 
+class Options
+{
+	private:
+	//different parameters of the options 
+	std::string short_command, long_command, short_description, long_description; 
+	
+	public:
+	//for only single long command without other arguments etc.like only port
+	Options(std::string long_command);
```

Comment:
parameter name can be command,  will help user to pass short as well as long.. May be use can use - and -- to distinguish them respectively

Reply:
As discussed we are going to drop this  facility of explicitly one parameter  in the options.